docs/developer.rst

```diff
@@ -36,6 +36,59 @@ You will also need the following modules:
   py2app (Mac)
   py2exe (Windows)
 
+Continuous Integration
+^^^^^^^^^^^^^^^^^^^^^^
+
+The main GitHub repository runs test on both the Travis (Linux) and AppVeyor
+(Windows) continuous integration platforms.
+
+Pull requests submitted to the repository will automatically be tested using
+these systems and results reported in the `checks` section of the pull request
+page.
+
+Services Used
+-------------
+
+Check the services' home pages in case you have to add the same capabilities to your master branch.
+
+::
+
+  Travis
+  AppVeyor
+  readthedocs
+  codecov
+
+
+links `Read The Docs`_, `codecov`_, `AppVeyor`_ and  `Travis`_
+
+After installing the servers we need to configure them by granting privileges to our testing user.
+
+For MySQL specifically, you need to create a new user called travis, and also a new database called testdb. This new user should be separate from the default MySQL user of root. After this is fulfilled, we can run the following commands to grant privileges. 
+
+::
+
+  MySQL
+  -----
+  GRANT ALL PRIVILEGES ON testdb.* TO 'travis'@'localhost';
+  GRANT FILE ON *.* TO 'travis'@'localhost';
+  ​
+  Install MySQL on Mac
```

Comment:
SO I need to test this on a new system. In postgres we basically first create a new user by default `postgres`  with password `testpass`,
`psql -c "CREATE USER postgres WITH PASSWORD 'testpass'"`.
then we create a db `testdb` and grant all privileges to the user on testdb.
In MYSQL, we assume that running the two commands does the same

```
 GRANT ALL PRIVILEGES ON testdb.* TO 'travis'@'localhost';
 GRANT FILE ON *.* TO 'travis'@'localhost';

```
but I think the case may be different. On @zhangcandrew's machine the same failed and I think I had this issue once. We are still trying to figure it out.

On that note @zhangcandrew, could you try to run this command 
https://github.com/weecology/retriever/blob/master/appveyor.yml#L60
`mysql -e "CREATE USER 'travis'@'localhost';GRANT ALL PRIVILEGES ON *.* TO 'travis'@'localhost';GRANT FILE ON *.* TO 'travis'@'localhost';" -uroot`